### PR TITLE
BaSP-MR-328: Fix table

### PR DESCRIPTION
**Pull request by FrancoLelli**
I fixed the table, so that when it is an array, the first element is displayed and then the remaining length.

- [X] create some way to show all members from table.

### CODE REVIEW

**Comment by FrancoLelli** [comment]
[fix table subscription](https://trello.com/c/2Jz0yQOV/328-fix-table-subscription)

**Review by Emanuel-Lamberti — APPROVED** [review]

File: src/Components/Shared/Table/index.js [hunk 1]
```diff
@@ -37,11 +37,24 @@ const TableComponent = ({
...
       if (Array.isArray(item)) {
-        return item?.map((content, contentIndex) => (
+        /* return item?.map((content, contentIndex) => (
           <span key={contentIndex}>
+            {console.log(content[fieldValue?.arrayFirstValue])}
             {content[fieldValue?.arrayFirstValue]} {content[fieldValue?.arraySecondValue]}
           </span>
-        ));
+        )); */
+        console.log(item);
```
Comment (maurojjzz) [inline src/Components/Shared/Table/index.js:46]: remove this

File: src/Components/Shared/Table/index.js [hunk 2]
```diff
@@ -37,11 +37,24 @@ const TableComponent = ({
   const ifArray = (item) => {
     if (item) {
       if (Array.isArray(item)) {
-        return item?.map((content, contentIndex) => (
+        /* return item?.map((content, contentIndex) => (
```
Comment (maurojjzz) [inline src/Components/Shared/Table/index.js:40]: remove commented lines

**Review by maurojjzz — APPROVED** [review]

**Review by CristianLotorto — APPROVED** [review]
great work! fix the comments

**Pull request merged**